transport_nantes/topicblog/views.py

```diff
@@ -681,6 +681,12 @@ class TopicBlogLauncherEdit(TopicBlogBaseEdit):
 
 
 class TopicBlogLauncherView(TopicBlogBaseView):
+    """This view is a bit curious.  In real life, TBLauncher's will be
```

Comment:
![image](https://user-images.githubusercontent.com/70256364/157059439-4b6c16ea-1197-4743-a9c7-940c534927b5.png)

It seems that the display is broken

Reply:
I thought that launchers had images ? In place of "SEE OUR NEW LAUNCHER!"

Reply:
In my test, I didn't upload an image.  I guess I'm taking on faith that the image works, I shouldn't do that. ;-)